Source: Elias Pipping <pipping@exherbo.org>
Upstream: reported as https://sourceforge.net/tracker/?func=detail&aid=3404133&group_id=4470&atid=104470
Reason: Do not use AM_CFLAGS for CXXFLAGS (includes -std=gnu99 which clang++ rejects)
--- devil-1.7.8/lib/Makefile.am	2009-03-08 21:45:36.000000000 +0100
+++ devil-1.7.8/lib/Makefile.am	2011-09-05 01:10:40.665927616 +0200
@@ -41,7 +41,7 @@
 
 libIL_la_CPPFLAGS = -I $(il_include) -I $(all_include)
 libIL_la_CFLAGS = $(AM_CFLAGS) @IL_CFLAGS@
-libIL_la_CXXFLAGS = $(AM_CFLAGS) @IL_CFLAGS@
+#libIL_la_CXXFLAGS = $(AM_CFLAGS) #@IL_CFLAGS@
 
 libIL_la_SOURCES = $(il_src)/il_alloc.c $(il_src)/il_doom.c $(il_src)/il_stack.c $(il_src)/il_pcx.c $(il_src)/il_rle.c $(il_src)/il_bits.c $(il_src)/il_endian.c $(il_src)/il_exr.cpp $(il_src)/il_jpeg.c $(il_src)/il_pic.c $(il_src)/il_sgi.c $(il_src)/il_bmp.c $(il_src)/il_error.c $(il_src)/il_lif.c $(il_src)/il_png.c $(il_src)/il_states.c $(il_src)/il_convbuff.c $(il_src)/il_files.c $(il_src)/il_main.c $(il_src)/il_pnm.c $(il_src)/il_targa.c $(il_src)/il_wdp.c $(il_src)/il_convert.c $(il_src)/il_gif.c $(il_src)/il_manip.c $(il_src)/il_profiles.c $(il_src)/il_tiff.c $(il_src)/il_cut.c $(il_src)/il_header.c $(il_src)/il_mdl.c $(il_src)/il_quantizer.c $(il_src)/il_utility.c $(il_src)/il_dcx.c $(il_src)/il_icon.c $(il_src)/il_mng.c $(il_src)/il_raw.c $(il_src)/il_wal.c $(il_src)/il_dds.c $(il_src)/il_internal.c $(il_src)/il_pal.c $(il_src)/il_rawdata.c $(il_src)/il_devil.c $(il_src)/il_io.c $(il_src)/il_pcd.c $(il_src)/il_register.c $(il_src)/il_psd.c $(il_src)/il_fastconv.c $(il_src)/il_neuquant.c $(il_src)/il_dds-save.c $(il_src)/il_psp.c $(il_src)/il_pix.c $(il_src)/il_pxr.c $(il_src)/il_vtf.c $(il_src)/il_xpm.c $(il_src)/il_hdr.c $(il_src)/il_icns.c $(il_src)/il_jp2.c $(il_src)/altivec_common.c $(il_src)/altivec_typeconversion.c $(il_src)/il_sun.c $(il_src)/il_nvidia.cpp $(il_src)/il_wbmp.c $(il_src)/il_squish.cpp $(il_src)/il_iff.c $(il_src)/il_ilbm.c $(il_src)/il_size.c $(il_src)/il_tpl.c $(il_src)/il_fits.c $(il_src)/il_iwi.c $(il_src)/il_blp.c $(il_src)/il_ftx.c $(il_src)/il_dicom.c $(il_src)/il_rot.c $(il_src)/il_texture.c $(il_src)/il_dpx.c $(il_src)/il_utx.cpp $(il_src)/il_mp3.c
 
